shard-operator-parent/shard-operator-core/src/main/java/com/redhat/service/smartevents/shard/operator/core/resources/CustomResourceStatus.java

```diff
@@ -39,10 +39,15 @@ public final Optional<Condition> getConditionByType(final String conditionType)
     }
 
     @JsonIgnore
-    public final boolean isReady() {
+    public final boolean isReady() { // Don't use this in V2 as READY constant doesn't exist
```

Comment:
follow up ticket for refactoring?

Reply:
https://issues.redhat.com/browse/MGDOBR-1369

Reply:
https://issues.redhat.com/browse/MGDOBR-1369